### PR TITLE
feat: add Selu activation operator support

### LINKED ISSUE #113: Add Selu activation function
### Feature description

Implement the Selu activation function in burn-onnx to follow the recently completed implementation in burn.

### Feature motivation

This addition will ensure models requiring Selu can be supported/exported to ONNX via burn-onnx, maintaining feature parity.

### (Optional) Suggest a Solution

Leverage the implementation details from burn, making any adjustments for ONNX workflow.

### PR DESCRIPTION
**Pull request by antimora**
## Summary

- Add dedicated `SeluProcessor` in onnx-ir with `alpha` and `gamma` config (ONNX spec defaults)
- Add `NodeCodegen` in burn-onnx using `burn::tensor::activation::selu(input)`
- Remove Selu from the fallback `ElementwiseUnaryProcessor` (now empty, cleaned up to only `ElementwiseBinaryNode`)
- Add integration test with positive and negative inputs

Closes #113

### CODE REVIEW

File: crates/burn-onnx/src/burn/node/selu.rs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        &self.outputs
+    }
+
+    fn forward(&self, scope: &mut ScopeAtPosition<'_>) -> TokenStream {
+        let input = scope.arg(self.inputs.first().unwrap());
+        let output = arg_to_ident(self.outputs.first().unwrap());
+
+        quote! {
+            let #output = burn::tensor::activation::selu(#input);
+        }
```
[the remark this answers is not in the document]
Comment (antimora) [reply crates/burn-onnx/src/burn/node/selu.rs:36]: Good catch. Added validation in codegen that panics if alpha or gamma differ from the ONNX defaults, since Burn's `selu()` hardcodes these constants. This matches the established pattern of rejecting unsupported features at codegen time per project conventions.

File: crates/burn-onnx/src/burn/node/selu.rs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    fn test_selu_forward() {
+        let node = create_node("selu1");
+        let code = codegen_forward_default(&node);
+        assert_snapshot!(code, @r"
+        pub fn forward(&self, input: Tensor<B, 2>) -> Tensor<B, 2> {
+            let output = burn::tensor::activation::selu(input);
+            output
+        }
+        ");
+    }
```
[the remark this answers is not in the document]
Comment (antimora) [reply crates/burn-onnx/src/burn/node/selu.rs:67]: Added `#[should_panic]" tests for both custom alpha and custom gamma, verifying the codegen rejects non-default values with clear error messages.
